state_machine/applications/system-check/tests/sensor_rate_test.py

```diff
@@ -0,0 +1,43 @@
+import time
+try:
+    from ulab.numpy import array, median
+except ImportError:
+    from numpy import array, median
+from lib.pycubed import cubesat
+
+def sensor_rate_test(sensor):
+    trials = 100
+    li = [0.0] * trials
+    dt_list = array(li)
```

Comment:
Minor, but `numpy.zeros(trials)` will also work here and is a little clearer of what's going on.